lib/ecto/adapters/sqlite3/connection.ex

```diff
@@ -332,11 +332,11 @@ defmodule Ecto.Adapters.SQLite3.Connection do
     end
   end
 
-  defp build_explain_query(query, :query_plan) do
+  def build_explain_query(query, :query_plan) do
     IO.iodata_to_binary(["EXPLAIN QUERY PLAN ", query])
   end
 
-  defp build_explain_query(query, :instructions) do
+  def build_explain_query(query, :instructions) do
     IO.iodata_to_binary(["EXPLAIN ", query])
   end
```

Comment:
Can both of these be left private?

Reply:
These changes are from #101, cc @jeregrine

Reply:
I don't remember exactly why I made them public, but I also don't feel like there is any reason to have private functions in elixir lol. Not like we are hiding data or encapsulating an API. Almost all end users will never use this api.

Reply:
I don't really agree with that, but it's a topic for a different day.